tide/master/PixelStreamWindowManager.cpp

```diff
@@ -111,10 +111,13 @@ void PixelStreamWindowManager::openWindow( const QString& uri,
 
     const auto type = _isPanel( uri ) ? ContentWindow::PANEL :
                                         ContentWindow::DEFAULT;
-
-    ContentPtr content = webbrowser ?
-                ContentFactory::getWebbrowserContent( uri ) :
-                ContentFactory::getPixelStreamContent( uri );
+    ContentPtr content;
+    if (stream == PixelStreamWindowManager::WEBBROWSER){
```

Comment:
this is javascript code style, does not match c++ style, but just leave it as it was:
ContentPtr content = (stream == PixelStreamWindowManager::WEBBROWSER) ? ContentFactory::getWebbrowserContent( uri ) :
ContentFactory::getPixelStreamContent( uri );